### ISSUE TITLE
Bug: v0.10 prints content with fields.values()

### ISSUE THREAD

**Issue by bryceglarsen** [tableau/document-api-python#240]
Hello!
It seems that a bug was introduce in the latest version of the Document API.
When running something such as:
`  datasource = Datasource.from_file(ds_file)
    for field in datasource.fields.values():
        print(field.name.rstrip("]").lstrip("["))`

we get content printed to terminal.

**Comment by bryceglarsen** [comment]
I believe it's: https://github.com/tableau/document-api-python/blob/master/tableaudocumentapi/field.py#L73

**Comment by bcantoni** [comment]
Yes that line was added in an earlier change and then fixed in #238. I'll leave this open until the next release comes out, then that problem should be resolved.

**Comment by bryceglarsen** [comment]
Ah, thanks @bcantoni! Guess I won't open a PR then. Heh.

**Comment by bcantoni** [comment]
@bryceglarsen this fix should be available in release 0.11 from this morning.

**Comment by bryceglarsen** [comment]
Thank you, @bcantoni!
